grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -122,7 +127,8 @@ class Order implements Serializable {
             "canceled",
             "completed",
             'activeOrderAdjustments',
-            'activeOrderItems'
+            'activeOrderItems',
+            'applicationEventPublisher'
```

Comment:
Are you getting a warning about this? I don't know if we want to mark this as transient because it's actually a property that has an implicit getter and setter. Most transients are derived properties that only have a getter.

Reply:
Exactly - when I hadn't marked it as transient, it wanted to be persisted to DB and the application didn't want to run.